utxo/utxo.go

```diff
@@ -1073,6 +1077,9 @@ func (uv *UtxoVM) undoTxInternal(tx *pb.Transaction, batch kvdb.Batch) error {
 		if bytes.Equal(addr, []byte(FeePlaceholder)) {
 			continue
 		}
+		if big.NewInt(0).SetBytes(txOutput.Amount).Cmp(big.NewInt(0)) == 0 {
```

Comment:
line 1080 and line 1079 do the same convert,` big.NewInt(0).SetBytes(txOutput.Amount)`,